jobs/src/main/scala/dcos/metronome/scheduler/impl/NotifyOfTaskStateOperationStep.scala

```diff
@@ -34,7 +33,8 @@ class NotifyOfTaskStateOperationStep(eventBus: EventStream, clock: Clock) extend
     (taskChanged.stateOp, taskChanged.stateChange) match {
 
       // A Mesos status update disregarding the effect
-      case (TaskStateOp.MesosUpdate(task, WithMesosStatus(mesosStatus), _), _) => Some(TaskState(mesosStatus))
+      case (TaskStateOp.MesosUpdate(task, marathonTaskStatus, mesosStatus, _), _) => Some(TaskState(mesosStatus))
```

Comment:
Seems fine :) Get rid of the comment below?

Reply:
marathonTaskStatus is not used, so "_" would be enough...

Reply:
maybe in the future we can use MarathonTaskStatus and get rid of Metronome mapping... ?

I think that what is in TaskState in metronome is probably very similar to what is happening in MarathonTaskStatus?